packages/accordion/test/accordion-panel.test.js

```diff
@@ -0,0 +1,142 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import { sendKeys } from '@web/test-runner-commands';
+import sinon from 'sinon';
+import '../vaadin-accordion-panel.js';
+
+describe('vaadin-accordion-panel', () => {
+  let panel;
+
+  describe('opened', () => {
+    let contentPart, contentNode;
+
+    beforeEach(async () => {
+      panel = fixtureSync(`
+        <vaadin-accordion-panel>
+          <div>Content</div>
+        </vaadin-accordion-panel>
+      `);
+      await nextRender();
+      contentPart = panel.shadowRoot.querySelector('[part="content"]');
+      contentNode = panel.querySelector('div');
+    });
+
+    it('should set opened to false by default', () => {
+      expect(panel.opened).to.be.false;
+    });
+
+    it('should reflect opened property to attribute', () => {
+      panel.opened = true;
+      expect(panel.hasAttribute('opened')).to.be.true;
+    });
+
+    it('should hide the content when opened is false', () => {
+      expect(getComputedStyle(contentPart).display).to.equal('none');
+    });
+
+    it('should show the content when `opened` is true', () => {
+      panel.opened = true;
+      expect(getComputedStyle(contentPart).display).to.equal('block');
+    });
+
+    it('should set aria-hidden on the slotted element to true by default', () => {
+      expect(contentNode.getAttribute('aria-hidden')).to.equal('true');
+    });
+
+    it('should set aria-hidden on the slotted element to false when opened', () => {
+      panel.opened = true;
+      expect(contentNode.getAttribute('aria-hidden')).to.equal('false');
+    });
```

Comment:
**note:** These can be replaced with snapshot tests at some point.

Reply:
Will do in a separate PR.